<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_raster_dialog</class>
 <widget class="QDialog" name="import_raster_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="importRasterComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Import from QGIS Layer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Import from Dataset</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Import from Desktop</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="confirmOK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
